packages/react-ui-ag/src/Listings/Listing.js

```diff
@@ -31,7 +31,12 @@ export default class Listing extends Component {
     theme: PropTypes.object,
     className: PropTypes.string,
     onClick: PropTypes.func,
-    photos: PropTypes.object,
+    photos: PropTypes.arrayOf(
```

Comment:
This isn't what you think it is.. https://github.com/rentpath/react-ui/pull/365/files#diff-be83dd49bf57212006170863d5504aa5L86

We should probably just rename this prop to `photoConfig` or something else.

Reply:
yeah makes more sense. Doesn't make sense anymore to construct the url in react-ui.

Reply:
And I went redo crazy before I realized. This is super WIP.